website/pages/docs/k8s/helm.mdx

```diff
@@ -182,6 +182,25 @@ and consider if they're appropriate for your deployment.
 
       - `secretKey` ((#v-global-cakey-secretkey)) (`string: null`) - The key of the Kubernetes secret.
 
+  - `lifecycleSidecarContainer` ((#v-global-lifecycle)) - The lifecycle sidecar ensures the Consul services
+    are always registered with their local Consul clients and is used by the ingress/terminating/mesh gateways
+    as well as with every Connect-injected service.
+
+    - `resources` ((#v-global-lifecycle-resources)) (`map`) - The resource requests and limits (CPU, memory, etc.)
+      for each of the lifecycle sidecar containers. This should be a YAML map of a Kubernetes
+      [ResourceRequirements](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) object. See values file for defaults.
```

Comment:
Should we say that defaults are shown below for this and others since we're including an example?

Reply:
If we claim that the example has the defaults, we'll need to update them any time we make a change. The example is to show the desired format only, since we previously had other resource settings take in a multi-line string instead of a map.